vic/drivers/image/src/vic_force.c

```diff
@@ -220,6 +321,17 @@ vic_force(void)
                                                 param.SNOW_MAX_SNOW_TEMP,
                                                 &(atmos[i].prec[j]), 1);
         }
+        // Validate vegcover
+        for (v = 0; v < options.NVEGTYPES; v++) {
+            vidx = veg_con_map[i].vidx[v];
+            if (vidx != -1) {
+                for (j = 0; j < NF; j++) {
+                    if (veg_hist[i][vidx].vegcover[j] < MIN_VEGCOVER) {
+                        veg_hist[i][vidx].vegcover[j] = MIN_VEGCOVER;
+                    }
+                }
+            }
+        }
```

Comment:
Yes, it is because the line I commented on (337) hasn't changed, therefor the diff didn't change.  If I had highlighted one of the lines that you changed, the diff would have changed (actually it would be collapsed and labeled as outdated).